README.md

```diff
@@ -0,0 +1,26 @@
+# Documentation Overview
+
+**[View the API Documentation](https://automattic.github.io/Gravatar-SDK-Android/current/index.html)**
```

Comment:
⛏️  Maybe we can link the root path in case, at some point, we change something.

```suggestion
**[View the API Documentation](https://automattic.github.io/Gravatar-SDK-Android**
```